flocker_bb/builders/flocker_vagrant.py

```diff
@@ -494,7 +579,9 @@ def getSchedulers():
     for distribution in ('fedora-20', 'centos-7', 'ubuntu-14.04'):
         builders = [
             configuration.builder_name
-            for configuration in ACCEPTANCE_CONFIGURATIONS
+            for configuration in (
+                ACCEPTANCE_CONFIGURATIONS +
```

Comment:
:question: Change this variable to something which is explicitly testing the node.